examples/methods/tram.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
```

Comment:
this needs to be renamed to `plot_tram.py` and adhere the style of the other examples (with the ReST block) so that it is picked up by sphinx gallery.